src/components/Footer.tsx

```diff
@@ -0,0 +1,69 @@
+import React, { useContext } from 'react';
+import { Filter } from '../types/Filter';
+import { TodosContext } from '../context/TodosContext';
+
+export const Footer: React.FC = () => {
+  const {
+    value: { todos },
+    filter: { statusFilter, setStatusFilter },
+    onClearCompleted,
+    completedCount,
+  } = useContext(TodosContext)!;
+
+  const notCompletedTodos = todos.filter(
+    (todo: { completed: unknown }) => !todo.completed,
+  );
+
+  const filterLinks = [
+    {
+      name: 'All',
+      href: '#/',
+      dataCy: 'FilterLinkAll',
+      value: Filter.All,
+    },
+    {
+      name: 'Active',
+      href: '#/active',
+      dataCy: 'FilterLinkActive',
+      value: Filter.Active,
+    },
+    {
+      name: 'Completed',
+      href: '#/completed',
+      dataCy: 'FilterLinkCompleted',
+      value: Filter.Completed,
+    },
+  ];
```

Comment:
Don't need to create array manually. Use `Object.values(Filter)` for creating array instead